relayer-test/src/chain/builder.rs

```diff
@@ -26,7 +26,7 @@ pub struct ChainBuilder {
     */
     pub command_path: String,
 
-    /*
+    /**
```

Comment:
Shouldn't these comments start with `///` ?

Reply:
`/**` also starts a doc comment block: https://doc.rust-lang.org/reference/comments.html

Reply:
Oh I didn't know about that, thanks! Any reason for picking block comments over line comments here?

Reply:
It is more convenient to always use multi-line comments so that when additional documentation needs to be added in the future, there is no need to re-edit the first line to use multi-line comment or use `///` on every line.